core-bundle/contao/themes/flexible/styles/layout/filter.pcss

```diff
@@ -17,7 +17,7 @@
 }
 
 .tl_panel:first-of-type {
-    border-radius: var(--border-radius) var(--border-radius) 0 0;
+    border-radius: calc(var(--border-radius) * 3) calc(var(--border-radius) * 3) 0 0;
```

Comment:
I agree that this makes sense in this example

<img width="480" height="445" alt="" src="https://github.com/user-attachments/assets/328f59b5-dab1-4002-bb23-15fb79e28d12" />

but not in our case, where the border radius of the field matches the one of the container exactly:

<img width="720" height="502" alt="" src="https://github.com/user-attachments/assets/737fac43-e414-42f5-9452-52bc9061ab6f" />